runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -326,7 +326,7 @@ Java_com_ibm_oti_vm_VM_getClassNameImpl(JNIEnv *env, jclass recv, jclass jlClass
 				utfData[arity] = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(leafComponentType->arrayClass->romClass))[1];
 			} else {
 				/* The / to . conversion is done later, so just copy the RAW class name here */
-				utfData[arity] = 'L';
+				utfData[arity] = 'L'; /* TODO: ValueTypes Once there are more specs for Java API Q types, this will need to be updated */
```

Comment:
Can you create an issue for this TODO and add it to the comment?  It makes it much easier to remember the TODO later

Reply:
Issue created:
https://github.com/eclipse/openj9/issues/4083